### ISSUE TITLE
Count Pages

### ISSUE THREAD

**Issue by therpobinski** [vizicsaba89/pdf-signer#29]
Is it possible to know how many pages the PDF has with the library that is used? It is necessary as I need to know how many pages there are so that I can implement the annotation on all sheets.

On the other hand, I have fixed library conflicts with large PDF files and different certificates. This is in this [pull](https://github.com/vizicsaba89/pdf-signer/pull/28). Hope they merge it soon.

**Comment by pankucsi** [comment]
We have used it for only small PDF files (1-4 pages).

I have merge your pull request, thanks for contributing. Can I ask you to extend the test cases with a large pdf and with different cerificates ? It would be very helpful.

**Comment by therpobinski** [comment]
Of course, I could test with large PDFs, but it is difficult for me to test with another type of certificate as I work with certificates from real people and I cannot filter this information.

On the other hand, I am very interested to know if it is possible to know how many sheets there are in a pdf. With this library.
